src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import Header from './Header';
+
+const Categories = () => (
+  <>
+    <Header />
```

Comment:
- [ ] great job so far.. Everything is perfect. Kindly ensure the categories page should display only a button with text "Check status"

Reply:
@tufoinnkuo10 

Hi Thank you for your review, the document says

> Should have a form for adding a book (no functionality yet).
> Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
> Categories:
> Should display a button "Check status" only.
> Styling is not required at this point.

https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/project_init_project_wth_comps_v2_0.md

Waiting for your valuable reply

Thank You

Reply:
@tufoinnkuo10 

Thank you for your review, added it now.

![image](https://user-images.githubusercontent.com/40719899/190488168-61dfe868-a545-4efc-8e6a-dda62ff1ca1b.png)